docs/pages/installation.mdx

```diff
@@ -151,13 +151,13 @@ supported under Windows 10 1607+ as of Teleport v7.2. We support running
     $ curl https://get.gravitational.com/teleport-v(=teleport.version=)-windows-amd64-bin.zip.sha256
     # <checksum> <filename>
     $ curl -O teleport-v(=teleport.version=)-windows-amd64-bin.zip https://get.gravitational.com/teleport-v(=teleport.version=)-windows-amd64-bin.zip
-    $ echo %PATH% # Edit %PATH% if necessary
+    $ echo $Env:Path # Edit PATH if necessary
```

Comment:
I'm also wondering if it might be a good idea to link instructions for editing system environment variables in Windows.
Maybe https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-7.2#using-systemenvironment-methods ?

Reply:
That would be a nice addition, I think (though not required)